sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessor.java

```diff
@@ -24,20 +25,22 @@
 import reactor.core.scheduler.Schedulers;
 
 /**
- * This is the starting point for event processor.
+ * Event Processor provides a convenient mechanism to consume events from all partitions of an Event Hub in the context
+ * of a consumer group. Event Processor based application consists of one or more instances of EventProcessor which are
+ * set up to consume events from the same Event Hub, consumer group to balance the workload across different instances
+ * and track progress when events are processed. Based on the number of instances running, each Event Processor may own
+ * 0 or more partitions to balance the workload among all the instances.
+ *
  * <p>
- * Event Processor based application consists of one or more instances of {@link EventProcessor} which are
- * set up to consume events from the same Event Hub + consumer group and to balance the workload across different
- * instances and track progress when events are processed.
+ * Event Processor based application consists of one or more instances of {@link EventProcessor} which are set up to
+ * consume events from the same Event Hub + consumer group and to balance the workload across different instances and
```

Comment:
Don't use '+' when you mean to use 'and'. Also, this paragraph could do with a comma or two.